src/tests/end-to-end/common/element-identifiers/details-view-selectors.ts

```diff
@@ -76,6 +76,8 @@ export const overviewSelectors = {
     outcomeSummaryBar: getAutomationIdSelector(outcomeSummaryBarAutomationId),
     testOutcomeChips: (testName: string) =>
         getAutomationIdSelector(testSummaryStatusAutomationId(testName)) + ' .outcome-chip',
+    loadAssessmentDialog: '.ms-Dialog-inner',
```

Comment:
We generally prefer not to use CSS styles for this. In this case specifically, these are also styles that we do not own. I would find a way to use data-automation-id here.

Reply:
I was able to put a data-automation-id on the button, this has been updated!